### PR TITLE
Toggles default based on value or selects other

### PR DESCRIPTION
**Pull request by straleyb**

### CODE REVIEW

**Comment by straleyb** [comment]
This chooses the default value if no value exists or if the default text is detected. Otherwise it chooses other and prefills with the proper value.

File: app/views/hyrax/base/_form_warning.html.erb [hunk 1]
```diff
@@ -14,14 +14,15 @@
 <ul>
   <li class="radio">
     <label>
-      <%= f.radio_button :content_alert, t('hyrax.content_warning.content_alert') %>
+      <% @checked = (f.object.content_alert.blank? || (f.object.content_alert == t('hyrax.content_warning.content_alert'))) %>
```
Comment (CGillen) [inline app/views/hyrax/base/_form_warning.html.erb:17]: Can you move the `@checked` definition out of the `<ul>` so it's clearer where it's going to be used?

File: app/views/hyrax/base/_form_warning.html.erb [hunk 2]
```diff
@@ -14,14 +14,15 @@
...
+      <%= f.radio_button :content_alert, t('hyrax.content_warning.content_alert'), class='content_alert_box', checked: @checked ? 'checked' : '' %>
```
Comment (CGillen) [inline app/views/hyrax/base/_form_warning.html.erb:18]: You probably already tested this, but does the `checked` form option work w/ regular truthy values? or does it automatically just set it to whatever is passed in?
`checked: @checked` would be cleaner, or `checked: !!@checked` if need be, but might not be possible all together

Comment (CGillen) [reply app/views/hyrax/base/_form_warning.html.erb:18]: `<input ... checked="checked">` is the most correct, but not required (except for xHTML), `<input ... checked>` or w/ any value is sufficient

**Review by CGillen — CHANGES_REQUESTED** [review]

File: app/views/hyrax/base/_form_warning.html.erb (hunk 2 above)
Comment (straleyb) [reply app/views/hyrax/base/_form_warning.html.erb:18]: I havent tested to see if the simple form options can just push through the `checked` html. I can see, but I know that it passes the checked in as an option. I can give that a bit of attention to see if I can get that working and update here.

**Review by CGillen — APPROVED** [review]

**Pull request merged**